PWGDQ/Tasks/tableReader.cxx

```diff
@@ -348,6 +369,197 @@ struct AnalysisMuonSelection {
   PROCESS_SWITCH(AnalysisMuonSelection, processDummy, "Dummy function", false);
 };
 
+struct AnalysisPrefilterSelection {
+  Produces<o2::aod::DalitzBitsReduced> dalitzbits;
+
+  // Configurables
+  Configurable<std::string> fConfigEventCuts{"cfgEventCuts", "eventStandardNoINT7", "Event selection"};
+  Configurable<std::string> fConfigDalitzTrackCuts{"cfgDalitzTrackCuts", "", "Dalitz track selection cuts, separated by a comma"};
+  Configurable<std::string> fConfigDalitzPairCuts{"cfgDalitzPairCuts", "", "Dalitz pair selection cuts"};
```

Comment:
We should use just one single pair cut for now i think.
It is fine to have several cuts in the Dalitz tagger for skimming, but here i think we should keep just one. It would be very simple then to apply these cuts later on in the downstream task, just via filter expressions, and without needing any further complication of the code.

Reply:
Indeed, this will be simpler, but with less possibility of comparison between prefilters or with/without. But maybe comparison is not really needed.
However I think in the prefilterselection if we want QA we might want to put multiple cuts for the histograms. Maybe in the QA case we can put as dalitzBit for prefiltering a single boolean which would be an OR on all trackCuts/pairCuts decision.